libs/ai-endpoints/langchain_nvidia_ai_endpoints/_common.py

```diff
@@ -561,7 +567,18 @@ def _postprocess_args(cls, values: Any) -> Any:
                         raise ValueError(
                             f"Model {name} is unknown, check `available_models`"
                         )
-
+        else:
+            if not name:
+                # set default model
+                name = values.get("client").default_model.id
+                values["model"] = name
+                warnings.warn(
```

Comment:
good that you're notifying the user about the action you're taking on their behalf.